libs/go/libp2p_node/dht/dhtpeer/dhtpeer.go

```diff
@@ -808,20 +846,29 @@ func (dhtPeer *DHTPeer) handleNewDelegationConnection(conn net.Conn) {
 			}()
 		}
 		// add to queue (nonblocking - buffered queue)
+		dhtPeer.syncMessagesLock.Lock()
+
 		select {
 		case dhtPeer.syncMessages[pair] <- envel:
 		default:
 			// send back! error
 			fmt.Println("CHANNEL FULL, DISCARDING <<<-------- ", string(envel.Message))
 		}
+		dhtPeer.syncMessagesLock.Unlock()
 	}
 
+	linfo().Str("addr", addr).
+		Msg("111 delegate client disconnected")
```

Comment:
why `111`?

Reply:
fixed. was done for debug. cause eyecatching